pkg/controller/masteruserrecord/sync.go

```diff
@@ -53,6 +53,10 @@ func (s *Synchronizer) synchronizeSpec() error {
 		s.memberUserAcc.Spec.UserAccountSpecBase = s.recordSpecUserAcc.Spec.UserAccountSpecBase
 		s.memberUserAcc.Spec.Disabled = s.record.Spec.Disabled
 		s.memberUserAcc.Spec.UserID = s.record.Spec.UserID
+		if s.memberUserAcc.Spec.UserID == "" {
+			s.memberUserAcc.Spec.UserID = s.recordSpecUserAcc.Spec.UserID
+		}
+		s.memberUserAcc.Spec.UserID = s.record.Spec.UserID
```

Comment:
Remove line 59.

Reply:
oh no! thanks @alexeykazakov forgot to remove that